File: Demo/试用IQKeyboardManager/试用IQKeyboardManager/ViewController.m
//
//  ViewController.m
//  试用IQKeyboardManager
//
//  Created by kadis on 2019/8/14.
//  Copyright © 2019 https://github.com/GGTechnology. All rights reserved.
//

#import "ViewController.h"
#import <IQKeyboardManager.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    //  键盘不显示 Toolbar
    [IQKeyboardManager sharedManager].enableAutoToolbar = NO;
    //  点击其它收起键盘
    [IQKeyboardManager sharedManager].shouldResignOnTouchOutside = YES;
}


@end

File: Demo/SWIFT/Swift_Block_Delegate/Swift_Block_Delegate/ViewController.swift
//
//  ViewController.swift
//  Swift_Block_Delegate
//
//  Created by kadis on 2020/6/5.
//  Copyright © 2020 lvguangda. All rights reserved.
//

import UIKit

class ViewController: UIViewController, delegateViewDelegate {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        let vD = delegateView.init(frame: CGRect(x: 100, y: 100, width: 100, height: 100));
        vD.delegate = self
        self.view.addSubview(vD)
        
        let bD = blockView.init(frame: CGRect(x: 100, y: 300, width: 100, height: 100));
        self.view.addSubview(bD)
        
//        bD.blockproerty = {(parameter) ->() in
//            self.view.backgroundColor = .yellow
//        }
        // 可以参略写成这样
        bD.blockproerty = {(parameter) in
            self.view.backgroundColor = .yellow
        }
 
    }
    
    func btnClick(tag: Int) {
        print(tag)
        self.view.backgroundColor = .green
    }
}

